_posts/news/2015-05-12-announcing-scalajs-0.6.3.md

```diff
@@ -0,0 +1,157 @@
+It is meant to define a subset of JavaScript that can be optimized more predictably by runtimes, and which also causes less surprises to developers.
+Contrary to [asm.js](http://asmjs.org/), it is still meant to be written by hand.
+
+Early experiments show that Strong Mode can be an excellent target for Scala.js, because their semantics and restrictions align in many ways.
+We have therefore invested time in providing a Strong Mode-compliant output to Scala.js.
+Similarly to the ECMAScript 6 output, it can be activated with an sbt setting:
+
+    scalaJSOutputMode := org.scalajs.core.tools.javascript.OutputMode.ECMAScript6StrongMode
+
+Since Strong Mode is a subset of ECMAScript 6, any runtime which supports ES6 can also run Strong Mode code.
+But to enable the additional checks, you have to use the `--strong-mode` argument to V8.
+Using io.js, this can be done with:
+
+    postLinkJSEnv := NodeJSEnv(executable = "iojs", args = Seq("--harmony-rest-parameters", "--strong-mode")).value
+    scalaJSStage := FastOptStage
+
+The same caveats as the ES6 mode apply.
+
+Scala.js is an excellent platform for experimenting with Strong Mode, as we can easily generate huge amounts of Strong Mode-compliant code just by recompiling applications in this mode.
+Since Strong Mode is still in flux, we are discussing with the V8 team to make it as good as it can be, for JavaScript developers at large as well as for Scala.js and compiler writers specifically.
+
+## Bug fixes
+
+Among others, the following bugs have been fixed:
+
+* [#1490](https://github.com/scala-js/scala-js/issues/1490) JVM style module initialization under semantic flag
+* [#1512](https://github.com/scala-js/scala-js/issues/1512) Atomic write of `-fastopt.js` and `-opt.js`
+* [#1556](https://github.com/scala-js/scala-js/issues/1556) Test adapter provisions slaves too aggressively
+* [#1560](https://github.com/scala-js/scala-js/issues/1560) `java.io.StringReader.read(c:Array[Char],offset:Int,len:Int)` returns 0 at eof, instead of -1
+* [#1564](https://github.com/scala-js/scala-js/issues/1564) Allow overriding path prefixes
+* [#1569](https://github.com/scala-js/scala-js/issues/1569) AssertionError when crossProject is used as a ProjectRef in another project
+* [#1578](https://github.com/scala-js/scala-js/issues/1578) Report the full stack trace for exceptions thrown in Futures
+* [#1582](https://github.com/scala-js/scala-js/issues/1582) Parsing hexadecimal Long that has 0x80000000 bit set fails
+* [#1589](https://github.com/scala-js/scala-js/issues/1589) Switch statement with "if" causes compiler to fail
+* [#1556](https://github.com/scala-js/scala-js/issues/1556) Test adapter provisions slaves too aggressively
+* [#1625](https://github.com/scala-js/scala-js/issues/1625) Emit warning if js.Dynamic.literal.applyDynamicNamed has duplicate keys (and [#1595](https://github.com/scala-js/scala-js/issues/1595) not emit illegal JS in that situation)
```

Comment:
Use `js.Dynamic.literal.applyDynamicNamed` instead of js.Dynamic.literal.applyDynamicNamed.